kadmin/del.c

```diff
@@ -64,9 +64,8 @@ do_del_ns_entry(krb5_principal nsp, void *data)
     char *unsp = NULL;
 
     if (krb5_principal_get_num_comp(context, nsp) != 2) {
-        (void) krb5_unparse_name(context, nsp, &unsp);
-        krb5_warn(context, ret = EINVAL, "Not a valid namespace name %s",
-                   unsp ? unsp : "<Out of memory>");
+        ret = krb5_unparse_name(context, nsp, &unsp);
```

Comment:
This change looks bogus, the old code was better. `unsp` isn't being used and the error is the result of `krb5_unparse_name()`, it should be `EINVAL`.

Reply:
Or, at least, is the `krb5_unparse_name()` call required?